acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/privacy/PrivacyNode.java

```diff
@@ -96,7 +96,8 @@ public PrivacyNode(final PrivacyNodeConfiguration privacyConfiguration, final Ve
             besuConfig.isRevertReasonEnabled(),
             besuConfig.getPlugins(),
             besuConfig.getExtraCLIOptions(),
-            new ArrayList<>());
+            new ArrayList<>(),
```

Comment:
Although `new ArrayList` exists before your changes, a nice pattern is to use `Collections.emptyList()` if the intent is to pass an empty list without the implementation details.